roles/etcd/tasks/main.yml

```diff
@@ -15,6 +15,9 @@
 - include: refresh_config.yml
   when: is_etcd_master
 
+# Reload systemd before starting service
+- meta: flush_handlers
```

Comment:
It's in a separate commit. When using reset.yml and then cluster.yml afterwards, it for some reason fails in the next task as it can not find the etcd service.